gravitee-apim-console-webui/src/management/api/reporter-settings/reporter-settings-message/reporter-settings-message.component.ts

```
@@ -365,4 +371,23 @@ export class ReporterSettingsMessageComponent implements OnInit {
       return null;
     };
   }
+
+  isGreaterThanMaxRate = (): ValidatorFn | null => {
+    return (control: AbstractControl): ValidationErrors | null => {
+      const value = control.value;
+      const limit = this.settings?.logging?.messageSampling?.windowedCount?.limit;
+      try {
+        const valueWindowedCount = WindowedCount.parse(value);
+        const limitWindowedCount = WindowedCount.parse(limit);
+        if (valueWindowedCount.rate() > limitWindowedCount.rate()) {
+          control.markAsTouched();
+          return { maxRate: `Windowed Count rate sampling should be less than ${limitWindowedCount}` };
+        }
+      } catch (error) {
+        control.markAsTouched();
+        return null;
+      }
```

At least add a console.error(error).
Else it's a silent failure.
You could also show a message to the user.